app/views/home.php

```diff
@@ -38,7 +38,7 @@
                 <div class="stickyCard">
                     <h2> <i class="fa-solid fa-paperclip"></i> Sticky Cocktail</h2>
                     <a href="/cocktails/<?= htmlspecialchars($stickyCocktail->getCocktailId()) ?>-<?= urlencode($stickyCocktail->getTitle()) ?>">
-                        <h3 class="stickyTitle"><?php echo htmlspecialchars($stickyCocktail->getTitle()); ?></h3>
+                        <h3 class="stickyTitle"><?= ucwords(strtolower($stickyCocktail->getTitle())); ?></h3> 
```

Comment:
use this instead
<?= htmlspecialchars(ucwords(strtolower($stickyCocktail->getTitle()))); ?>